app/R/plot_doubling_distribution.R

```diff
@@ -0,0 +1,21 @@
+#' Function to plot discretised duration distribution
+#'
+#' This function provides custom plots for the output of `distcrete`.
+#'
+#' @param x a `numeric` vector of doubling times
+#' @param title a string to be added to the resulting ggplot
+#' 
+#' @author Sam Clifford
+#' 
+
+plot_doubling_distribution <- function(x, title = NULL) {
+  dat <- data.frame(x = x)  
+  ggplot2::ggplot(data = dat,
+                  ggplot2::aes(x = x)) +
+    ggplot2::geom_histogram(fill = cmmid_color, col = "white") +
```

Comment:
maybe specify bins or binwidth here, to silence the warning.

Reply:
Yep doing it right now.

Reply:
Done as of 119827237a7486b6fa9f2d1160387d637a82e2ea